docs/sources/review/lint-prose/index.md

```diff
@@ -59,3 +59,68 @@ The task title contains a verb and an object. For example: "Create a dashboard".
 
 <!-- vale Grafana.Quotes = YES -->
 ```
+
+## Use Vale in VS Code
+
+You can use Vale to lint your current document in VS Code.
+
+1. Clone the [Technical Documentation](https://github.com/grafana/technical-documentation) repository.
+
+   ```bash
+   git clone git@github.com:grafana/technical-documentation.git
+   ```
+
+1. Download [Vale](https://github.com/errata-ai/vale/releases).
```

Comment:
I'm not sure if this got lost in the fray or if you intended to remove this comment, but I definitely stared at this page not knowing what to download for a while. I don't think it hurts to point out that folks should look at the Assets list and find the one that's for them (already a bit of a step since I had to do further Googling to figure out which one was good for my Mac).